charts/internal/cilium-monitoring/values.yaml

```diff
@@ -0,0 +1,69 @@
+allowedMetrics:
+  # allowed metrics for cilium-agent
+  ciliumAgent:
+  - cilium_process_virtual_memory_bytes
+  - cilium_process_resident_memory_bytes
+  - cilium_process_open_fds
+  - cilium_bpf_maps_virtual_memory_max_bytes
+  - cilium_bpf_progs_virtual_memory_max_bytes
+  - cilium_agent_api_process_time_seconds_count
+  - cilium_agent_api_process_time_seconds_sum
+  - cilium_bpf_syscall_duration_seconds_count
+  - cilium_bpf_syscall_duration_seconds_sum
+  - cilium_bpf_map_ops_total
+  - kvstore_operations_total
+  - cilium_kvstore_operations_duration_seconds_sum
+  - cilium_kvstore_events_queue_seconds_count
+  - cilium_forward_count_total
+  - cilium_forward_bytes_total
+  - cilium_datapath_conntrack_gc_entries
+  - cilium_ip_addresses
+  - cilium_datapath_errors_total
+  - cilium_services_events_total
+  - cilium_unreachable_health_endpoints
+  - cilium_unreachable_nodes
+  - cilium_unreachable_health_endpoints
+  - cilium_drop_count_total
+  - cilium_nodes_all_events_received_total
+  - cilium_drop_bytes_total
+  - cilium_nodes_all_num
+  - cilium_policy_l7_denied_total
+  - cilium_policy_l7_forwarded_total
+  - cilium_policy_l7_received_total
+  - cilium_policy_l7_parse_errors_total
+  - cilium_proxy_upstream_reply_seconds_sum
+  - cilium_proxy_upstream_reply_seconds_count
+  - cilium_triggers_policy_update_call_duration_seconds_sum
+  - cilium_policy_endpoint_enforcement_status
+  - cilium_proxy_redirects
+  - cilium_triggers_policy_update_total
+  - cilium_policy_count
+  - cilium_policy_import_errors
+  - cilium_policy_max_revision
+  - cilium_endfpoint_regeneration_time_stats_seconds_bucket
+  - cilium_endpoint_regenerations
+  - cilium_endpoint_state
+  - cilium_controllers_runs_total
+  - cilium_controllers_failing
+  - cilium_controllers_runs_duration_seconds_sum
+  - cilium_controllers_runs_duration_seconds_count
+  - cilium_k8s_client_api_latency_time_seconds_sum
+  - cilium_k8s_client_api_latency_time_seconds_count
+  - cilium_k8s_client_api_calls_counter
+  - cilium_kubernetes_events_received_total
+  - cilium_kubernetes_events_total
+  - cilium_process_cpu_seconds_total
+  - cilium_errors_warnings_total
+  - cilium_endpoint_regeneration_time_stats_seconds_bucket
+
+  hubble:
+  - hubble_flows_processed_total
+  - hubble_drop_total
+  - hubble_port_distribution_total
+  - hubble_tcp_flags_total
+  - hubble_icmp_total
+
+  # allowed metrics for cilium-operator
+  ciliumOperator:
```

Comment:
ciliumOperator metrics are not being scraped by prometheus. Target is up but the metrics do not exist in prometheus